python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -164,8 +183,22 @@ def createTempExp(self, calexpRefList, skyInfo, visitId=0):
                 calExpRef = calExpRef.butlerSubset.butler.dataRef("calexp", dataId=calExpRef.dataId,
                                                                   tract=skyInfo.tractInfo.getId())
                 calExp = self.getCalExp(calExpRef, bgSubtracted=self.config.bgSubtracted)
-                exposure = self.warpAndPsfMatch.run(calExp, modelPsf=modelPsf, wcs=skyInfo.wcs,
-                                                    maxBBox=skyInfo.bbox).exposure
+                covName = 'covar'
+                for key, val in calExpRef.dataId.items():
+                    covName += '_%s_%s'%(key, val)
```

Comment:
I've added a small amount of extra functionality to store the name of the covariance matrix in the the CovView object but I think that until I have the covariance matrix for the full visit, I'm going to defer figuring out how to persist the covariance matrices for the individual exposures.